### PR TITLE
fix chars release

### PR DESCRIPTION
**Pull request by s00d**

### CODE REVIEW

**Comment by cawa-93** [comment]
Could you provide more details about the problem and how you solved it.

**Comment by s00d** [comment]
I have a symbol "	" in the history of changes and publication falls. you need to delete it.
. I also advise you to change everything " to ', there were also problems with json parse

File: scripts/release-notes.js [hunk 1]
```diff
@@ -53,7 +53,9 @@ async function getCommits() {
       const jsonReady = `[${commitsArray
         .map(rawCommitText => {
           return rawCommitText
+            .replace(/	/g, ' ')
             .replace(/"/g, '\\\"')
+            .replace(/'/g, '\\\"')
```
Comment (cawa-93) [inline scripts/release-notes.js:58]: Would you please provide samples of problem commits, and how does this change fix them?

Comment (s00d) [reply scripts/release-notes.js:58]: [1.txt](https://github.com/cawa-93/vite-electron-builder/files/6222236/1.txt)
unfortunately the repository is private, here is the log. problems start on 2 line

Comment (s00d) [reply scripts/release-notes.js:58]: [1.txt](https://github.com/cawa-93/vite-electron-builder/files/6222265/1.txt)
with problem quotes

Comment (cawa-93) [reply scripts/release-notes.js:58]: I still don't see the need to replace single quotes `'` with double quotes `"`.

I tried to reproduce your problem. Without replacing quotes, everything works as expected:
```

    Merge remote-tracking branch 'remote/main' into test-local

    # Conflicts:
    #       .github/FUNDING.yml
    #       .github/workflows/release.yml
    #       README.md
    #       electron-vendors.config.js
    #       package-lock.json
    #       package.json
    #       packages/renderer/vite.config.js
```
```markdows
### Other Changes
- Merge remote-tracking branch 'remote/main' into test-local. (d08582b)
  # Conflicts:
  #  .github/FUNDING.yml
  #  .github/workflows/release.yml
  #  README.md
  #  electron-vendors.config.js
  #  package-lock.json
  #  package.json
  #  packages/renderer/vite.config.js
```

File: scripts/release-notes.js [hunk 2]
```diff
@@ -53,7 +53,9 @@ async function getCommits() {
       const jsonReady = `[${commitsArray
         .map(rawCommitText => {
           return rawCommitText
+            .replace(/	/g, ' ')
```
Comment (cawa-93) [inline scripts/release-notes.js:56]: I think it is better to replace the tab character with **2 spaces** to preserve the visual formatting of the text.

File: scripts/release-notes.js (hunk 1 above)
Comment (s00d) [reply scripts/release-notes.js]: ok, cleaned it up, I didn't really understand why I had such a problem.

**Review by cawa-93 — APPROVED** [review]

**Comment by cawa-93** [comment]
@s00d, could you test the latest version of the [`scripts/release-notes.js`](https://github.com/cawa-93/vite-electron-builder/blob/57501ddb89b4611675203f8c9eddec5bcdad598b/scripts/release-notes.js) in your repository? I changed the algorithm and stopped using JSON.

**Pull request merged**